styles/octocat.css

```diff
@@ -11,3 +11,73 @@ ul {
 * {
   box-sizing: border-box;
 }
+
+img {
+  max-width: 100%;
+}
+
+strong {
+  color: black;
+}
+
+.prompt {
+  text-align: center;
+}
+
+.section {
+  padding: 7rem;
```

Comment:
Let the padding left and right be a smaller value so that the content can be closer to the edges of the viewport on mobile.